### DISCUSSION TITLE
Community

### DISCUSSION THREAD

**Discussion by A1-exe** [Alic3C/Cyber-FastTrack-Spring-2021#21] [General]
Is there a discord or some platform I can join to connect with other participants?

**Comment by JoshHeng** [comment]
Of course, come and join us at https://discord.cyberdiscoverycommunity.uk!

We're a friendly, growing community of students taking part in Cyber Discovery and SANS' CyberStart family of programmes. Our community is mainly centered around a discord server consisting of over 4000+ members from a multi-talented, diverse range of backgrounds.

Although the focus of our server is cyber security and learning based, our members carry out insightful discussions on a range of subjects including STEM fields, media, gaming and computer hardware. Whether you're looking for pointers, hints or guidance on particular challenges, student run capture the flag events or more. It's a great place to hang out casually!

**Reply by RealJammy** [reply]
There's another server called RACTF too!

**Reply by JoshHeng** [reply]
> 
> 
> There's another server called RACTF too!

What's RACTF?

**Reply by RealJammy** [reply]
Have you heard of RACTF? It’s the latest, most mind-bogglingly awesome, breathtakingly stunning capture the flag competition this side of the pond. We have so much awesomeness prepared it’s practically bursting at the seams with its awesomeness. Don’t remember to take part in this year’s totally epic, government-approved, really awesome ctf! Don’t forget to follow us on twitter at @RACTF_UK!

**Reply by 404dcd** [reply]
Wait, what? There's a CTF run entirely by CD people?

**Reply by RealJammy** [reply]
Yes, all CD people! Some of them are even cyberists!
